codalab/apps/web/templates/web/competitions/dumps.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
+{% block page_title %}Competition Dumps{% endblock %}
+{% block head_title %}Competition Dumps{% endblock %}
+
+{% block content %}
+    <a href="{% url "competitions:view" pk=competition.pk %}" >Go Back to Competition Homepage</a>
+    <div class="panel">
+        <div class="panel-body">
+            <button id="make_dump" class="btn btn-primary">Create Competition Dump</button>
+            {% if dumps %}
+                <h1>Dumps for: {{ competition.title }}</h1>
+                <table width="80%" style="border: solid rgba(0,0,0,0.31) 1px;" class="table table-responsive">
+                <thead>
+                    <td width="100px"><h5><b>Download link</b></h5></td>
+                    <td width="100px"><h5><b>Status</b></h5></td>
+                    <td width="400px"><h5><b>Publish Date</b></h5></td>
+                </thead>
+                    <tbody>
+                        <tr style="padding: 20px; margin: 20px;">
+                            {% for dump in dumps %}
+                                {% if forloop.counter0|divisibleby:1 %}<tr>{% endif %}
+                                <td>
+                                    {% if dump.status == "Finished" %}
+                                        <a href="{{ dump.sassy_url }}">Download</a>
+                                    {% endif %}
+                                </td>
+                                <td>
+                                    <b>{{ dump.status }}</b>
+                                </td>
+                                <td>
+                                    <i>{{ dump.timestamp }}</i>
+                                </td>
+                                {% if forloop.counter|divisibleby:1 %}</tr>{% endif %}
+                            {% endfor %}
+                        </tr>
+                    </tbody>
+                </table>
+            {% else %}
+                <i>No dumps found for the current competition.</i>
+            {% endif %}
+        </div>
+    </div>
+    <script>
+        $(document).ready(function() {
+            $("#make_dump").click(function() {
+                var url = "{% url 'competitions:make_bundle' competition_pk=competition.pk %}"
+                $.post(url)
+                    .success(function() {
+                        alert("Bundle is being created, please refresh the page to view new dumps.")
```

Comment:
Let's just refresh the page here, I think you can just do that with `window.location = window.location.href` or something?